app/presenters/two_factor_auth_code/max_attempts_reached_presenter.rb

```diff
@@ -22,6 +22,23 @@ def header
     end
 
     def description
```

Comment:
I know it's part of the interface with the parent class, but I wish that we could rename this to something plural since it's retuning an array

Reply:
> I know it's part of the interface with the parent class, but I wish that we could rename this to something plural since it's retuning an array

Well... to create another wrinkle, _technically_ I've refactored it so that it could return a string _or_ an array, even though it's only currently using the array supports.

I could definitely see a YAGNI argument to picking the one we need, though it felt a bit more the exception that description would need to be multiple paragraphs.

Reply:
> Yup I saw the `Array()` in the template, the refactor needed to support separate `description` and `description_parts` methods are not worth it to me

Just to clarify, are you saying... ?

- Keep it as it is proposed currently
- Or: Only support what we need (i.e. arrays)
   - ...and name accordingly?
- Or: Something else?

Reply:
Sorry, that was unclear. I'm saying: 
> Keep it as it is proposed currently
I was just grumbling at the interface we've defined in the superclass, because this one case is not enough reason to rename and change the whole interface

Reply:
> I was just grumbling at the interface we've defined in the superclass, because this one case is not enough reason to rename and change the whole interface

Yeah, then again, we only have two classes which implement this base class 😄  Though based on what I was seeing with #5941, it seems like it could be a handy abstraction.